dts/bindings/sensor/ti,ina237.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2021 Grinn
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
```

Comment:
This is duplicating common properties with ti,ina230.yaml. I think this should use a common include file instead. See dts/bindings/serial/nordic,nrf-uart-common.yaml and the places that include it for an example.

Reply:
Fixed.